### ISSUE TITLE
Unable to focus on the clear mark for the search field

### ISSUE THREAD

**Issue by wonko-gsa** [GSA/sam-design-system#1405]
**Components Involved**
sds-search

**Expected Behavior**
A User can focus on the clear mark in the search field by a tab key 

**Actual Behavior**
A User cannot focus on the clear mark in the search field by a tab key. 

**Your Environment**
Angular Version: 16.2.12

@gsa-sam packages version:
```
    "@gsa-sam/components": "16.0.11",
    "@gsa-sam/layout": "16.0.8",
    "@gsa-sam/layouts": "16.0.0",
    "@gsa-sam/ngx-uswds": "16.0.1",
    "@gsa-sam/ngx-uswds-icons": "16.0.0",
    "@gsa-sam/sam-formly": "16.0.11",
    "@gsa-sam/sam-material-extensions": "16.0.11",
    "@gsa-sam/sam-styles": "3.0.18",
    "@gsa-sam/shared": "16.0.8",
    "@ngx-formly/core": "6.2.1",
```

**Ticket Number**
N/A (as of 1/25/24)

**Additional context**
a. UI
![image](https://github.com/GSA/sam-design-system/assets/117218338/b2a86aeb-58c1-4c03-9789-099fff723660)

b. 
![image](https://github.com/GSA/sam-design-system/assets/117218338/320be2ae-474a-4838-94c8-b8a44604ddd5)

**Comment by yerramshilpa** [comment]
The issue was resolved in @gsa-sam/components@16.0.14 version

**Comment by yerramshilpa** [comment]
For the future reference, the original issue existing in the USWDS components, created an issue in the USWDS repo, here is the issue number
https://github.com/uswds/uswds/issues/5786
